### PR TITLE
Created the frontend for Login & Create Account pages

### PR DESCRIPTION
**Pull request by Theos1010**
Created Login Page for both Instructor and Student (Username, Password, Sign In and Create new account)
Created Register Page for both Instructor and Student (First name, Full name, Student ID, Username, Password, Sign Up and Sign In if already have an account)

### CODE REVIEW

File: CodebustersPAS-ClientApp/src/pages/LoginPage.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+              placeholder="Enter your password"
+            />
+          </div>
+
+          <button type="submit" className="sign-in-btn">Sign In</button>
+        </form>
+
+        <div className="sign-up-link">
+          <p>Don't have an account?</p>
+          <a href="/signup">Create one</a>
```
Comment (LouisChoiniere) [inline CodebustersPAS-ClientApp/src/pages/LoginPage.tsx]: You should use `onClick={() => navigate('/')}` like on the create account page

**Review by LouisChoiniere — CHANGES_REQUESTED** [review]
Fix comment otherwise looks good

File: CodebustersPAS-ClientApp/src/pages/CreateAccountPage.tsx [hunk 2]
```diff
@@ -65,12 +65,12 @@ const CreateAccount: React.FC = () => {
           </div>
 
           <div className="form-group">
-            <label htmlFor="fullName">Full Name</label>
+            <label htmlFor="lastName">Full Name</label>
```
Comment (Theos1010) [inline CodebustersPAS-ClientApp/src/pages/CreateAccountPage.tsx]: @tinaw1412 change full name to last name at line 68

File: CodebustersPAS-ClientApp/src/pages/CreateAccountPage.tsx [hunk 3]
```diff
@@ -65,12 +65,12 @@ const CreateAccount: React.FC = () => {
...
             <input
-              id="fullName"
+              id="lastName"
               type="text"
-              value={fullName}
-              onChange={(e: ChangeEvent<HTMLInputElement>) => setFullName(e.target.value)}
+              value={lastName}
+              onChange={(e: ChangeEvent<HTMLInputElement>) => setLastName(e.target.value)}
               required
               placeholder="Enter your full name"
```
Comment (Theos1010) [inline CodebustersPAS-ClientApp/src/pages/CreateAccountPage.tsx]: @tinaw1412 Change full name to last name at line 75

**Review by LouisChoiniere — APPROVED** [review]

**Review by tinaw1412 — APPROVED** [review]

**Pull request merged**